pyproject.toml

```diff
@@ -21,7 +21,7 @@ classifiers = [
 ]
 requires-python = ">=3.7"
 dependencies = [
-  "jupyter_ydoc>=0.1.16,<0.2.0",
+  "jupyter_ydoc>=0.2.0,<0.3.0",
```

Comment:
Maybe we could require `jupyter_ydoc>=0.2.0,<0.4.0`, and pin `jupyter_ydoc>=0.2.0,<0.3.0` in JupyterLab v3 and `jupyter_ydoc>=0.3.0a0` in JupyterLab v4?